static/app/api.tsx

```diff
@@ -499,7 +499,7 @@ export class Client {
           responseJSON,
           responseText,
           getResponseHeader: (header: string) => response.headers.get(header),
-          rawResponse: response.clone(),
+          rawResponse: response,
```

Comment:
Can you grep in project for `rawResponse` to make sure nothing downstream is trying to access its body?

Reply:
and now we don't `clone()` at all anymore, because i removed from the `rawResponse` from the returned object, so now we only ready the body once, try to parse it as json unconditionally (which could be better...) and that's it!